ontologies/Items.ttl

```diff
@@ -0,0 +1,161 @@
+	rdfs:range wi:Classmark ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+	
+wi:hasFileFormat rdf:type owl:Class ;
+	rdfs:label "hasFileFormat"@en ;
+	rdfs:comment "This relates an item to its file format when the item comprises a digital object."@en ;
+ 	rdfs:domain wi:Item ;
+	rdfs:range wi:FileFormat ;
+  rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+  
+#######  Data properties #####  
+	
+wi:identifier rdf:type owl:DatatypeProperty ;
+	rdfs:label "identifier"@en ;
+	rdfs:comment "Relates the item to an identifier that is preferred, either a unique system-generated identifier that governs interaction between systems, or one used for human citation, such as a manuscript number to be cited in footnotes."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .		
+  
+wi:barcode rdf:type owl:DatatypeProperty ;
+	rdfs:label "barcode"@en ;
+	rdfs:comment "Relates the item to a number string encoded on physical items by a barcode and used for the purposes of movement control."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:barcode rdf:type owl:DatatypeProperty ;
+	rdfs:label "barcode"@en ;
+	rdfs:comment "Relates the item to a number string encoded on physical items by a barcode and used for the purposes of movement control."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:boxNumber rdf:type owl:DatatypeProperty ;
+	rdfs:label "boxNumber"@en ;
+	rdfs:comment "Relates the item (generally an archive item) to a number identifying the box within which it is stored."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:outsizeItemNumber rdf:type owl:DatatypeProperty ;
+	rdfs:label "boxNumber"@en ;
+	rdfs:comment "Relates the item (generally an archive item) to a number identifying its storage location when the item is too large to fit into a standard Library box."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:physicalDescription rdf:type owl:DatatypeProperty ;
+	rdfs:label "physicalDescription"@en ;
+	rdfs:comment "A description of specific physical characteristics of the item - condition, dimensions, details of cover or binding, etc. - or, in the case of digital objects, any notes relating to this digital manifestation over and above notes of file format."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+  
+wi:sheetSize rdf:type owl:DatatypeProperty ;
+	rdfs:label "sheetSize"@en ;
+	rdfs:comment "A note of the width and length dimensions of a flat artwork."@en ;
```

Comment:
That's right, although as a free-text field I'm not sure that it would be consistent enough to do useful things with those numbers; indeed, it's even possible that we have mixed units.